lib/foods.js

```diff
@@ -1,5 +1,37 @@
+function persistFood(foodName, foodCalories) {
+  if (window.localStorage) {
+    var rawFoodObject = localStorage.getItem('foodObject');
+    if (rawFoodObject == null) {
+      rawFoodObject = "{}";
```

Comment:
i'd suggest storing this as an empty array, then adding javascript objects to the array. this way, we can easily iterate over the collection of food objects